### ISSUE TITLE
Pull in campaigns automatically

### ISSUE THREAD

**Issue by sbsmith86** [DoSomethingArchive/legacy-website#6428]
Pull in campaigns to the takeover screen automatically. As a user signs up for campaigns, they are removed and replaced with others. 
- The campaigns are pull in based on type, issue, source of traffic or some other category
  - Or some combination i.e a few collection campaigns, a few based on issue
- Only campaigns the users is not signed up for
- They should show up in random order.

**Comment by chloealee** [comment]
dosomething_campaign_preprocess_action_page

```
  $campaign_id = '123';
  $campaign_blurb = "blah blah";

  drupal_add_js(
    array('dosomethingCampaign' =>
      array(
        'recommended_campaigns' => array(
          'campaign_id' => $campaign_id,
          'blub' => $campaign_blurb,
        ),
      ),
    ),
    'setting'
  );

```

**Comment by chloealee** [comment]
Closed with commit 6344a5f. 

Uses existing `dosomething_campaign_get_recommended_campaign_nids` function to dynamically add campaigns. Only thing that is missing that is listed above is that campaigns are not replaced with others after a user signs up for campaign (instead, "SIGNUP" button just changes text to "Signed up!").
